### PR TITLE
Update install guide for conda

### LINKED ISSUE #562: Install with Conda instructions have a syntax problem
At
https://isis.astrogeology.usgs.gov/documents/InstallGuide/index.html

Where the instructions indicate to run isis3VarInit.py by specifying different arguments to isis3VarInit.py, the line reads:

`./$CONDA_PREFIX/scripts/isis3VarInit --data-dir=[path to data directory]  --test-dir=[path to test data directory]`

Which has a few of issues:
- It assumes a particular pwd
- the name of the script is missing its .py extension

### PR DESCRIPTION
**Pull request by ihumphrey**
This addresses #562

### CODE REVIEW

File: isis/src/docsys/documents/InstallGuide/InstallGuide.xml [hunk 1]
```diff
@@ -82,7 +82,7 @@ you must <em>not</em> upgrade the ISIS Data Files!!!
          and $ISIS3TESTDATA=$CONDA_PREFIX/testdata.  The user can specify different directories
          for both of these optional values:
 
-         <strong>./$CONDA_PREFIX/scripts/isis3VarInit --data-dir=[path to data directory]  --test-dir=[path to test data directory]</strong>
+         <strong>python $CONDA_PREFIX/scripts/isis3VarInit.py --data-dir=[path to data directory]  --test-dir=[path to test data directory]</strong>
```
Comment (jessemapel) [inline isis/src/docsys/documents/InstallGuide/InstallGuide.xml:85]: The script has a shebang, so you don't need python at the start

Comment (ihumphrey) [reply isis/src/docsys/documents/InstallGuide/InstallGuide.xml:85]: For my conda env, the scripts are all downloaded with 664 permissions set, so I can't actually run the script without using the python interpreter

Comment (jessemapel) [reply isis/src/docsys/documents/InstallGuide/InstallGuide.xml:85]: :+1:

**Comment by ihumphrey** [comment]
@jessemapel can this be merged?

**Pull request merged**